recipe/meta.yaml

```diff
@@ -60,7 +60,7 @@ requirements:
     - astropy >=5.0,<6.dev0
     - bokeh >=2.0,<3.dev0
     - ctapipe >=0.19.2,<0.20.dev0
-    - ctapipe_io_lst >=0.22.3,<0.23.dev0
+    - ctapipe_io_lst~=0.23.0
```

Comment:
I dont think the ~= notation works in conda recipes... this is why it was using the two part version before.

Reply:
No, it doesn't: https://conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#package-match-specifications

Reply:
we need the newest one, so:
`>=0.23.1,<0.24.dev0`
?

Reply:
It should be the same as the pypi package, which has what you put here: ` ~=0.23.0`, so the equivalent conda requirement is:

` >= 0.23.0,<0.24.dev0`